src/autoComplete/index.tsx

```diff
@@ -4,52 +4,49 @@ import cc from 'classcat'
 import isEmpty from 'lodash.isempty'
 import debounce from 'lodash.debounce'
 
-import prefix from '_utils'
 import TextField from 'textField'
 import { ItemChoiceStatus } from 'itemChoice'
 import AutoCompleteList from './autoCompleteList'
-import AutoCompleteListItemDefault from './autoCompleteListItemDefault'
 import style from './style'
 
 type query = string | number | boolean
 interface AutoCompleteProps {
-  readonly name: string
-  readonly isSearching: boolean
-  readonly searchForItems: (query: query) => void
-  readonly onInputChange?: (params: Partial<onChangeParameters>) => void
-  readonly searchForItemsMinChars?: number
-  readonly defaultValue?: string
-  readonly onSelect?: (obj: AutocompleteOnChange) => void
-  readonly onFocus?: (event: React.FocusEvent<HTMLInputElement>) => void
-  readonly onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void
-  readonly onClear?: () => void
-  readonly className?: Classcat.Class
-  readonly inputClassName?: Classcat.Class
-  readonly itemClassName?: Classcat.Class
-  readonly bodyClassName?: Classcat.Class
-  readonly items?: AutocompleteItem[]
-  readonly maxItems?: number
-  readonly renderItem?: (itemToRender: AutocompleteItemToRender) => React.ReactElement<any>
-  readonly renderBusy?: ({ query }: { query: query }) => React.ReactElement<any>
-  readonly renderNoResults?: ({ query }: { query: query }) => React.ReactElement<any>
-  readonly renderQuery?: (item: AutocompleteItem) => string
-  readonly renderEmptySearch?: AutocompleteItem[]
-  readonly getItemValue?: (item: AutocompleteItem) => string
-  readonly inputAddon?: React.ReactElement<any>
-  readonly placeholder?: string
-  readonly busyTimeout?: number
-  readonly debounceTimeout?: number
-  readonly autoFocus?: boolean
-  readonly focus?: boolean
-  readonly buttonTitle?: string
-  readonly showList?: boolean
-  readonly onDoneAnimationEnd?: () => void
-  readonly autoCorrect?: 'on' | 'off'
-  readonly disabled?: boolean
-  readonly readOnly?: boolean
-  readonly required?: boolean
-  readonly error?: string | JSX.Element
-  readonly selectedItemStatus?: ItemChoiceStatus
+  readonly name: string,
```

Comment:
Why do you have commas at the end of the line here when you don't have some in the ItemChoiceProps interface?

Reply:
Prettier doesn't work sometimes, don't know why